R/hypothesize.R

```diff
@@ -0,0 +1,93 @@
+# hypothesize.R
+#' @title hypothesize
+#' @author Nada Elnour, \email{nada.elnour@@mail.utoronto.ca}
+#'
+#' \code{hypothesize} generates an annotated hypothesis graph whose nodes are
+#' components of \code{mySys} and edges are GGI interpretations in either
+#' \code{EMAP} or \code{GMAP}
+#'
+#' @param network A dataframe of physically-interacting system components with
```

Comment:
As above: changing the parameters and results according to this format
https://github.com/hyginn/BCB420.2019.ESA/blob/master/PullRequestReviewNotes.md

Reply:
fixed